### PR TITLE
tplink device tracker documentation update

### PR DESCRIPTION
**Pull request by JackNova**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** [home-assistant/home-assistant#15356](https://github.com/home-assistant/home-assistant/pull/15356)

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/device_tracker.tplink.markdown [hunk 1]
```diff
@@ -13,7 +13,14 @@ ha_release: pre 0.7
 ---
 
 
-The `tplink` platform allows you to detect presence by looking at connected devices to a [TP-Link](https://www.tp-link.com) device. This includes the ArcherC9 line.
+The `tplink` platform allows you to detect presence by looking at connected devices to a [TP-Link](https://www.tp-link.com) device.
+
+Currently supported devices includes following:
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.tplink.markdown]: ... includes the following:

File: source/_components/device_tracker.tplink.markdown [hunk 2]
```diff
@@ -32,7 +39,7 @@ device_tracker:
 Configuration variables:
 
 - **host** (*Required*): The IP address of your router, e.g., 192.168.1.1.
-- **username** (*Required*: The username of an user with administrative privileges, usually *admin*.
+- **username** (*Required*): The username of an user with administrative privileges, usually *admin*. The ArcherD9 last firmware does not require a username
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.tplink.markdown]: ... username of a user... The Archer D9... does not require a username.

File: source/_components/device_tracker.tplink.markdown [hunk 3]
```diff
@@ -45,3 +52,8 @@ For Archer C9 models running firmware version 150811 or later please use the enc
 6. Copy the returned value to your Home Assistant configuration as password.
 
 See the [device tracker component page](/components/device_tracker/) for instructions how to configure the people to be tracked.
+
+For Archer D9 model the default ip is 192.168.1.1, the username is not necessary and you can leave that field blank
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.tplink.markdown]: ... and you can leave that field blank.

**Comment by frenck** [comment]
This PR is targeted against the wrong branch. It should go into `next`.

**Comment by frenck** [comment]
Current will be merged back into next pretty soon, so I'm going hold it off for a couple of days ;)

**Comment by frenck** [comment]
Changed base to `current`, since the parent PR has been released.
I'll go ahead and merge this one in.

**Review by frenck — APPROVED** [review]

**Pull request merged**